src/__tests__/Config.test.js

```diff
@@ -81,7 +81,7 @@ describe('writeConfigFile', () => {
     let json = { name: 'wat', version: '0.0.0' };
     let fileContents =
       JSON.stringify(json, null, 2).replace(/\n/g, '\r\n') + '\r\n';
-    let config = await Config.init(filePath);
+    let config = (await Config.init(filePath)) || {};
```

Comment:
Why `|| {}`?

Reply:
Since it is returning a maybe it needs a default.

Reply:
moved to throwing an error instead of a maybe and this got better.